<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="animated_btn1">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>110</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>parallel anim</string>
    </property>
   </widget>
   <widget class="QPushButton" name="animated_btn2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>230</y>
      <width>131</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>sequential anim</string>
    </property>
   </widget>
   <widget class="QPushButton" name="animated_btn3">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>310</y>
      <width>88</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>close</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>animated_btn3</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>226</x>
     <y>342</y>
    </hint>
    <hint type="destinationlabel">
     <x>321</x>
     <y>357</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
